test/types/copy-dir.d.ts

```diff
@@ -0,0 +1,13 @@
+declare module 'copy-dir' {
+  const copydir: {
```

Comment:
Are there no definitions in definitely typed for unixify / copy-dir? We could fix that! (I'm a maintainer of DT so I could get it merged pretty quick)

Reply:
There aren't - actually, I've never pushed something to DT, I usually just type the part of the API I need from the Documentation.
But it could be a nice exercise to do that for once :)

Reply:
If you wanted to raise a PR and mention me I can review it and get it merged. If you'd rather keep it in here for now that's fine. I don't want to boil the ocean 😀

Reply:
I'll add it to my TODO list :D